Owncloud iOs Client/Utils/Errors/ManageNetworkErrors.m

```
@@ -35,10 +35,7 @@ - (void)manageErrorHttp:(NSInteger)errorHttp andErrorConnection:(NSError *)error
     switch (errorConnection.code) {
         case kCFURLErrorUserCancelledAuthentication: { //-1012
             
-            [_delegate showError:NSLocalizedString(@"not_possible_connect_to_server", nil)];
-            
-            CheckAccessToServer *mCheckAccessToServer = [CheckAccessToServer new];
-            [mCheckAccessToServer isConnectionToTheServerByUrl:user.url];
+            [[CheckAccessToServer sharedManager] isConnectionToTheServerByUrl:user.url];
```

We should not hide error message from the user